### PR TITLE
Make SDK support optional

### PR DESCRIPTION
**Pull request by danielbreves**
:koala:

/cc @zendesk/quokka @ydrain @chucknado 

### Risks
 - None

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -4,32 +4,42 @@
...
         }
+      },
+      "zafSdkSupport": {
+        "label": {
+          "title": "A checkbox to enable ZAF SDK support. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo",
+          "value": "Enable ZAF SDK support?"
+        },
+        "helpText": {
+          "title": "Help text to explain the ZAF SDK support option. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo",
+          "value": "Check this box if the iframed website requires SDK support. For more information check out our developer documentation."
```
Comment (svizzari) [inline translations/en.json]: > ...requires __ZAF__ SDK support...

?

**Comment by svizzari** [comment]
Where did the new icon come from?

:+1: with one minor comment

**Comment by danielbreves** [comment]
@svizzari that's the icon currently used in production, but I think it was only ever updated in Monitor.

**Comment by svizzari** [comment]
> that's the icon currently used in production, but I think it was only ever updated in Monitor.

gotcha

**Comment by princemaple** [comment]
:+1:

**Comment by adammw** [comment]
Need to make the yml file changes too for translations to be swept correctly? Otherwise :+1: (although not entirely sure why you'd need to disable the sdk).

File: translations/en.json [hunk 2]
```diff
@@ -4,32 +4,42 @@
...
         }
+      },
+      "zafSdkSupport": {
+        "label": {
+          "title": "A checkbox to enable ZAF SDK support. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo",
+          "value": "Enable ZAF SDK support?"
+        },
+        "helpText": {
+          "title": "Help text to explain the ZAF SDK support option. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo",
+          "value": "Check this box if the iframed website requires ZAF SDK support. For more information check out our developer documentation."
```
Comment (chucknado) [inline translations/en.json]: Original: "For more information check out our developer documentation."

Suggestion: Include a URL to a specific location in the dev docs (a lot of content to navigate if you're not familiar with it!). Also, our "for more info" convention for UI strings is to use the string "Learn more" and apply the link to it.

File: translations/en.json [hunk 3]
```diff
@@ -4,32 +4,42 @@
...
+        }
       }
     },
     "description": {
-      "title": "Add a custom icon to the sidebar & embed any website, aka Sidebar Icon app. See https://zendesk.app.box.com/s/sn4u8kxq6k1zsy5nu66xhurobpbf9tac",
-      "value": "Add a custom icon to the sidebar & embed any website, aka Sidebar Icon app"
+      "title": "Embed any website in your Zendesk (formerly known as Sidebar Icon app). See https://zendesk.box.com/s/0c5a122j1hdgbx4yu6bqlktdtl1qfu03",
+      "value": "Embed any website in your Zendesk (formerly known as Sidebar Icon app)"
```
Comment (chucknado) [inline translations/en.json]: String: "Embed any website in your Zendesk (formerly known as Sidebar Icon app)"

Problem: This sentence is saying that _Zendesk_ was formerly known as the Sidebar Icon app. 
Not sure how much room you have to play with, but a suggested alternative would be: "Embed any website in your Zendesk. This app was formerly known as Sidebar Icon app."

**Comment by ydrain** [comment]
cc: @pdeuter @mariasica @injuhwang 
We will need to retranslate these strings.

**Comment by danielbreves** [comment]
@adammw to prevent issues like ZD#1052255 from happening in the future.

File: translations/en.json (hunk 2 above)
Comment (danielbreves) [inline translations/en.json]: That was my initial thought, but we escape links and we don't support markdown at the moment, so it looked pretty :hankey: 

screen shot 2015-04-16 at 10 56 45 am

File: translations/en.json (hunk 3 above)
Comment (danielbreves) [inline translations/en.json]: 80 characters :/

File: translations/en.json (hunk 2 above)
Comment (chucknado) [inline translations/en.json]: Maybe specify the doc to look for? Example: 'See the "Iframes in Apps" topic in our developer documentation". Or if you have more room: 'See the "Iframes in Apps" topic in the Zendesk Apps docs on https://developer.zendesk.com.'

File: translations/en.json (hunk 3 above)
Comment (chucknado) [inline translations/en.json]: OK, I checked the context (below). Maybe we can get away with just replacing the parentheses with a period. They're really two independent statements. Example:

"Embed any website in your Zendesk. Formerly known as Sidebar Icon app"

**Pull request merged**